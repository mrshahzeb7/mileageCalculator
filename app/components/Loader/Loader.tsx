import React from 'react';
import {ActivityIndicator, View, ViewStyle} from 'react-native';
import {colors} from '../../theme';

interface Props {}

export const Loader: React.FC<Props> = () => {
  return (
    <View style={$container}>
      <ActivityIndicator color={colors.primaryButton} size={'large'} />
    </View>
  );
};

const $container: ViewStyle = {
  flex: 1,
  alignItems: 'center',
  justifyContent: 'center',
};
