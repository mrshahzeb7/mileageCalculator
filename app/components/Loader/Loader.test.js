import 'react-native';
import React from 'react';
import {Loader} from './Loader';
import renderer from 'react-test-renderer';

describe('Loader', () => {
  it(' Loader renders correctly', () => {
    const tree = renderer.create(<Loader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
