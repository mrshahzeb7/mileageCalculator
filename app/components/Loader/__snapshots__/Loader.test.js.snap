// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader  Loader renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    color="#93b2fd"
    size="large"
  />
</View>
`;
